src/Playwright.Tests/DefaultBrowserContext1Tests.cs

```diff
@@ -249,7 +249,7 @@ public async Task ShouldSupportJavascriptEnabledOption()
 
         [PlaywrightTest("defaultbrowsercontext-1.spec.ts", "should support httpCredentials option")]
         [Test, Timeout(TestConstants.DefaultTestTimeout)]
-        public async Task ShouldRupportHttpCredentialsOption()
+        public async Task ShouldRepportHttpCredentialsOption()
```

Comment:
I don't think Repport is much better than Rupport, I guess you want Support.

Reply:
lol. true. fixed